advanced/is-pattern-1/README.md

```diff
@@ -17,7 +17,8 @@ steps provided in the following quick start guide.<br><br>
 
 * An already setup [Kubernetes cluster](https://kubernetes.io/docs/setup).<br><br>
 
-* Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/).<br><br>
+* Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/). Please note that Helm resources for WSO2 product
```

Comment:
@Chathuru no restriction on the NGINX Ingress Controller version. Please remove the version specification.

Reply:
Fix in https://github.com/Chathuru/kubernetes-is/commit/2b8053040bd2cdb86fac354216969387988a2d5d commit